<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1306</width>
    <height>735</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pb_send">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>580</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Отправить</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Chat_edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>811</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>555</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите сообщение:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="input_edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>811</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>811</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_nick">
     <item row="0" column="0">
      <widget class="QLabel" name="label_nick">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Введите ваш ник:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEdit_nick">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>60</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pb_nick">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Ввести</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>811</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_settings">
     <item row="3" column="1">
      <widget class="QSpinBox" name="spinBox_client_remote">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximum">
        <number>3000</number>
       </property>
       <property name="value">
        <number>1337</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pb_ok">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Ок</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBox_client_local">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximum">
        <number>3000</number>
       </property>
       <property name="value">
        <number>1337</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_loca_ip">
       <property name="text">
        <string>Введите ваш IP:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="lineEdit_remote">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>60</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEdit_local">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>127.0.0.1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_remote_ip">
       <property name="text">
        <string>Введите IP удаленного пользователя:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_port_local">
       <property name="text">
        <string>Ваш порт:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_port_remote">
       <property name="text">
        <string>Порт удаленного пользоваетля:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pb_recovery">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>550</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Восстановить предыдущую переписку</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_photo">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>580</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Отправить файл</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_ban">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>390</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>ЧС</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ban">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>390</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_ban">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>350</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите IP, кого хотите забанить:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_unban_">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>80</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите IP, кого хотите разбанить:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_unban">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>120</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_unban">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>120</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>ИЗ ЧС</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1306</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
